### PR TITLE
Implement placeholders without additional input

### LINKED ISSUE #25: Placeholders without additional input?
Would it be possible to add support for placeholders which are skipped when Tab-ing, so they can just receive their values from vimscript?
I'm giving minisnip a try (currently using Ultisnips).
When in Ultisnips I have the following snippet, I only have to tab once and I can immediately start typing the title

```
# `!v strftime("%Y-%m-%d %H:%M")` ${1:title}
```

However in minisnip I would have to tab twice with the following similar snippet:
```
# {{+~strftime("%Y-%m-%d %H:%M")+}} {{+title+}}
```

Maybe it could be a syntax such as:

     {{`strftime(...)`}}

### PR DESCRIPTION
**Pull request by thepith**
This should fix #25.

Snippets having `g:minisnip_skipmarker` (default is `` ` ``) just after the `g:minisnip_startdelim` are expanded, but not selected for further modification.

Examples:
```
{{+`~strftime("%Y-%m-%d")+}} {{++}}
```
and
```
{{+#+}}{{+`~\~1+}}{{+`~\~1+}}{{+`~\~1+}}{{+`~\~1+}}{{+`~\~1+}}
```

### CODE REVIEW

File: plugin/minisnip.vim [hunk 1]
```diff
@@ -10,6 +10,7 @@ let g:minisnip_trigger = get(g:, 'minisnip_trigger', '<Tab>')
 let g:minisnip_startdelim = get(g:, 'minisnip_startdelim', '{{+')
 let g:minisnip_enddelim = get(g:, 'minisnip_enddelim', '+}}')
 let g:minisnip_evalmarker = get(g:, 'minisnip_evalmarker', '~')
+let g:minisnip_skipmarker = get(g:, 'minisnip_skipmarker', '`')
```
Comment (joereynolds) [inline plugin/minisnip.vim]: How about we just use the evalmarker as the skipmarker?
Chances are you never want to purposely <tab> into an evalmarker's contents so we should skip it anyway.
Thoughts?

Comment (thepith) [reply plugin/minisnip.vim]: I sometimes use the backrefmarker to set a default value, which I might want to change. For example I use in latex:
```
\begin{figure}
  \centering
      \includegraphics[{{+width=\textwidth+}}]{{{+figure+}}}
  \caption{{{+Caption+}}}
  \label{fig:{{+~\~2+}
\end{figure}
```
Therefore I would prefer not to use the evalmarker as the skipmarker. I think having the possibility to explicitly decide whether it should be skipped or not is a good feature.

Perhaps we could do it the other way round. If there is the `evalmarker` the placeholder is skiped, unless a `donotskipmarker` is present?

Comment (joereynolds) [reply plugin/minisnip.vim]: I like the idea of a donotskip as I think by default the majority of eval snippets will be things you just want evaluating.

Thanks for the PR, looks good so far :)

**Comment by joereynolds** [comment]
Hey @thepith 

Nice work, works a treat and bonus points for writing tests too!

:D

**Comment by teranex** [comment]
Works perfectly. Nice. Thanks!

**Pull request merged**